src/libraries/System.Net.Sockets/src/Resources/Strings.resx

```diff
@@ -193,7 +194,7 @@
     <value>The specified value cannot be negative.</value>
   </data>
   <data name="ArgumentOutOfRange_Bounds_Lower_Upper" xml:space="preserve">
```

Comment:
Consider renaming this since it doesn't share a value with System.Private.Corelib.  e.g. `ArgumentOutOfRange_Bounds_Lower_Upper_Named`.